backend/app/api/endpoints/openapi_responses.py

```
@@ -112,7 +118,9 @@ def _task_to_response_object(
 
 
 @router.post("")
+@limiter.limit(settings.RATE_LIMIT_CREATE_RESPONSE)
 async def create_response(
+    request: Request,
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)